content/en/universal_service_monitoring/_index.md

```diff
@@ -19,22 +21,21 @@ further_reading:
   text: "Read about the Service Map"
 ---
 
-{{< beta-callout url="#" d-toggle="modal" btn_hidden="true" d_target="#signupModal" custom_class="sign-up-trigger">}}
-  Universal Service Monitoring (USM) is in public beta. There is currently no billing impact for enabling and using USM.
-{{< /beta-callout >}}
+## Overview
 
-Universal Service Monitoring (USM) provides visibility into your service health metrics universally across your entire stack _without having to instrument your code_. It relies solely on the presence of a configured Datadog Agent and [Unified Service Tagging][1], and brings performance data about your uninstrumented services into APM views such as the Services List, Service details, and Service Map. USM also works with [Deployment Tracking][2], Monitors, Dashboards, and SLOs.
+Universal Service Monitoring (USM) provides visibility into your service health metrics universally across your entire stack _without having to instrument your code_. It relies solely on the presence of a configured Datadog Agent and [Unified Service Tagging][1], and brings performance data about your uninstrumented services into APM views such as the Service Catalog, Services List, and Service Map. USM also works with [Deployment Tracking][2], Monitors, Dashboards, and SLOs.
 
-{{< img src="tracing/universal_service_monitoring/service_overview.mp4" alt="Video demonstrating Universal Service Monitoring. An overview of a service is accessed by clicking on a service on the Service Map and selecting View service overview." video="true" >}}
+{{< img src="universal_service_monitoring/service_overview.mp4" alt="Video demonstrating Universal Service Monitoring. An overview of a service is accessed by clicking on a service on the Service Map and selecting View service overview." video="true" >}}
 
 ### Supported versions and compatibility
 
 Required Agent version
-: Universal Service Monitoring requires that the Datadog Agent installed alongside your service be at least version 7.40.
+: Universal Service Monitoring requires that the Datadog Agent installed alongside your service be at least version 6.40/7.40.
 
 Supported platforms
 : Linux kernel 4.14 and greater<br/>
 CentOS or RHEL 8.0 and greater<br/>
+Windows IIS
 
 Supported application-layer protocols
 : HTTP<br/>
```

Comment:
For HTTPS (below this line), let's add GnuTLS alongside OpenSSL?

Reply:
```suggestion
: HTTP<br/>
HTTPS with GnuTLS or Open SSL
```
Is this what you mean, @sunhay ?

Reply:
Looks good!